inst/private/check_and_convert.m

```diff
@@ -0,0 +1,36 @@
+function obj = check_and_convert()
+  newl = sprintf('\n');
+  
+  str_check_is_sym = strjoin({'if temp is None or isinstance(temp, (sp.Basic, sp.MatrixBase)):',
```

Comment:
Why did you check for None in your code? I couldn't replicate a case to get None. I put it here just to follow your code.